misk/src/test/kotlin/misk/web/jetty/WebActionsServletTest.kt

```diff
@@ -0,0 +1,131 @@
+package misk.web.jetty
+
+import com.netflix.concurrency.limits.limit.SettableLimit
+import com.netflix.concurrency.limits.limiter.SimpleLimiter
+import misk.Action
+import misk.ApplicationInterceptor
+import misk.asAction
+import misk.client.UnixDomainSocketFactory
+import misk.inject.KAbstractModule
+import misk.testing.MiskTest
+import misk.testing.MiskTestModule
+import misk.time.FakeClock
+import misk.time.FakeClockModule
+import misk.web.*
+import misk.web.actions.LivenessCheckAction
+import misk.web.actions.ReadinessCheckAction
+import misk.web.actions.StatusAction
+import misk.web.actions.WebAction
+import misk.web.interceptors.UserInterceptorTest
+import misk.web.mediatype.MediaTypes
+import okhttp3.Headers
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.OkHttpClient
+import okhttp3.Protocol
+import okhttp3.Request
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import java.io.File
+import java.time.Duration
+import java.util.*
+import javax.inject.Inject
+
+@MiskTest(startService = true)
+class WebActionsServletTest {
+  @MiskTestModule
+  val module = TestModule()
+
+  @Inject
+  internal lateinit var jettyService: JettyService
+
+  private var socketName: String = "@udstest" + UUID.randomUUID().toString()
+
+  @Test
+  fun networkSocketSuccess() {
+    val response = get("/potato", SourceSocket.Type.NETWORK)
+    assertThat(response.code).isEqualTo(200)
+    assertThat(response.header("ActualSocketType")).isEqualTo("NETWORK")
+    assertThat(response.header("ActualSocketName")).isEqualTo("null")
+  }
+
+  @Test
+  fun udsSocketSuccess() {
+    val response = get("/potato", SourceSocket.Type.UNIX)
+    assertThat(response.code).isEqualTo(200)
+    assertThat(response.header("ActualSocketType")).isEqualTo("UNIX")
+    assertThat(response.header("ActualSocketName")).isEqualTo(socketName)
+  }
+
+  internal class WebActionsServletNetworkInterceptor : NetworkInterceptor {
+    override fun intercept(chain: NetworkChain) {
+      chain.httpCall.statusCode = 200
+      chain.httpCall.addResponseHeaders(
+          Headers.Builder()
+            .set("ActualSocketType", chain.httpCall.sourceSocket.type.toString())
+            .set("ActualSocketName", chain.httpCall.sourceSocket.name?:"null")
```

Comment:
Are you formatting the code with the Square styles imported into IJ?

Reply:
probably not

Reply:
Make sure to do that before making a non-draft PR. go/cashdevguide has instructions.

Reply:
The styles are imported, nothing changed 🦊